ui/src/widgets/ui-notification/UINotification.vue

```diff
@@ -98,7 +111,10 @@ export default {
         },
         close (payload) {
             this.show = false
-            this.$socket.emit('widget-action', this.id, payload)
+
+            const msg = this.messages[this.id] || {}
+            msg.payload = payload
```

Comment:
This is causing a very brief flash changed of the notification as it closes. We'd be better off here making a new `msg` object to emit, rather than overriding the local scope.

Reply:
Hey @joepavitt 
Thanks for reviewing this in depth!!
I hadn't noticed the brief flashing.  Will commit your changes...